.travis.yml

```diff
@@ -21,6 +21,10 @@ install:
   - 'ng version'
 
 before_script:
+  - pwd
+  - wget http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/2.2.3/swagger-codegen-cli-2.2.3.jar -O swagger-codegen-cli.jar
+  - java -jar swagger-codegen-cli.jar generate -i https://dockstore.org:8443/swagger.json -l typescript-angular2 -o src/app/shared/swagger
```

Comment:
This could get weird if the production site gets much behind ui2.
I wonder if this could just be pointed at a tagged version of https://github.com/ga4gh/dockstore/blob/develop/dockstore-webservice/src/main/resources/swagger.yaml